### ISSUE TITLE
Level: Saboteur

### ISSUE THREAD

**Issue by andybalaam** [andybalaam/rabbit-escape#527]
```
:name=Saboteur
:description=
:author_name=
:author_url=
:solution.1.code=#ETXKCK~hpTEoKzXCKvK$KzLEfGVoEC_Go%jEJTGv$T7JgL%%G}y#KAC
:num_rabbits=3
:num_to_save=3
:rabbit_delay=5
:bash=2
:bridge=1
:dig=1
#     Q  
    y#  #
  #######
   r     
         
  / MMMMM
O#b     M
MMMMMMMMM
```

https://github.com/andybalaam/rabbit-escape/blob/master/rabbit-escape-engine/src/rabbitescape/levels/staging/rabbots/Saboteur.rel

**Comment by andybalaam** [comment]
Feedback from @raory299 - this has an unintended solution where you bridge the saboteur rabbit off the right side of the level. This can be fixed with a higher metal wall on the right.

**Comment by andybalaam** [comment]
Hopefully removed trivial solutions.

**Comment by andybalaam** [comment]
Released in 0.11
